src/test/isolation/expected/create_index_hot_optimizer.out

```diff
@@ -0,0 +1,35 @@
+Parsed test spec with 2 sessions
+
+starting permutation: s2begin s2select s1optimizeroff s1update s1createindexonc s2select s2forceindexscan s2select s2abort s2forceindexscan s2expalin
+step s2begin: BEGIN ISOLATION LEVEL SERIALIZABLE;
+step s2select: select '#' as expected, c from hot where c = '#'
+                  union all
+                  select '$', c from hot where c = '$';
+expected       c              
+
+#              #              
+step s1optimizeroff: set optimizer = off;
+step s1update: update hot set c = '$' where c = '#';
+step s1createindexonc: create index idx_c on hot (c);
+step s2select: select '#' as expected, c from hot where c = '#'
+                  union all
+                  select '$', c from hot where c = '$';
+expected       c              
+
+#              #              
+step s2forceindexscan: set enable_indexscan=on; set enable_seqscan=off;
+step s2select: select '#' as expected, c from hot where c = '#'
+                  union all
+                  select '$', c from hot where c = '$';
+expected       c              
+
+#              #              
+step s2abort: abort;
+step s2forceindexscan: set enable_indexscan=on; set enable_seqscan=off;
+step s2expalin: explain (costs off) select c from hot where c = '$';
+QUERY PLAN     
+
+Gather Motion 3:1  (slice1; segments: 3)
+  ->  Index Scan using idx_c on hot
+        Index Cond: (c = '$'::bpchar)
+Optimizer: Pivotal Optimizer (GPORCA)
```

Comment:
May be it would be more concise if we return to the original test form and remove the _optimizer variant? Anyway there is no difference in the plan before and after the patch.

Reply:
we are not testing the plan itself, but the fact that the cache was reset after the transaction was completed.

Reply:
You are right. Then you should write enough comments describing the cache situation.

Reply:
it may really be worth removing this test since this logic is not covered for postgres. or maybe we should write a separate test to test the plan cache reset for postgres and orca. what do you think?

Reply:
How are we going to write a separate test case? To cover both planners we'll have to show only PREPARE commands

Reply:
we can try using PREPARE for select in this test. although it may be an unnecessary overcomplication.

Reply:
I think it's an overhead as well. Let's return to the first variant with `set optimizer = off` only.